FF150 Relnote: Sanitizer.replaceElementWithChildren() return false for <html> (#43670)
mdn/content: files/en-us/web/api/sanitizer/replaceelementwithchildren/index.md

### Return value

> `true` if the operation changed the configuration to set the element to be replaced by its children, and `false` otherwise.

Reviewer: ```suggestion
A boolean value: `true` if the operation updated the `Sanitizer` configuration to replace the element with its children, and `false` otherwise.
```

Reviewer: Specify it's a boolean, plus simplify the existing phrasing, which was rather fiddly to read.

> The method returns `false` if the sanitizer was already configured to replace the given element, or if the replacement element was {{htmlelement("html")}} in the HTML namespace.

Reviewer: ```suggestion
The method returns `false` if the sanitizer is already configured to replace the given element, or if the replacement element is {{htmlelement("html")}} in the HTML namespace.
```
